platform/ios/src/MGLAnnotationView.h

```diff
@@ -41,6 +41,19 @@ NS_ASSUME_NONNULL_BEGIN
 @property (nonatomic, assign, getter=isFlat) BOOL flat;
 
 /**
+ This property defaults to NO and becomes YES when the view is tapped/clicked on.
+ Selecting another view will first deselect the currently selected view.
+ */
+@property (nonatomic, assign, getter=isSelected) BOOL selected;
+- (void)setSelected:(BOOL)selected animated:(BOOL)animated;
```

Comment:
Document this method also, and mention that you shouldn’t call this method directly.